src/test/shell/bazel/cc_integration_test.sh

```diff
@@ -603,9 +603,8 @@ function test_cc_starlark_api_additional_inputs() {
   cat >> "$pkg"/BUILD << EOF
 cc_bin(
     name = "g",
-    srcs = ["e.cc"],
```

Comment:
Cannot the test pass with an already existing target? Why is this modification necessary?